flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -663,24 +660,12 @@ private void disposeSavepoint(ClusterClient<?> clusterClient, String savepointPa
 
 	protected void executeProgram(
 			final Configuration configuration,
-			final PackagedProgram program) throws ProgramMissingJobException, ProgramInvocationException {
+			final PackagedProgram program) throws ProgramInvocationException {
 		logAndSysout("Starting execution of program");
 
-		JobSubmissionResult result = ClientUtils.executeProgram(DefaultExecutorServiceLoader.INSTANCE, configuration, program);
-
-		if (result.isJobExecutionResult()) {
```

Comment:
We could print this information in the `ContextEnvironment.executeAsync()`, when the job completion future fires. I think users are used to seeing this information when running jobs using the CLI so not printing this would be confusing.

Reply:
Yes, I thought of printing information in execution. Will push a follow-up.

Reply:
Actually with my follow up behavior also changed because previously(current master) we print the last execution result here and if we move to context environment we print execution result per execution.

Reply:
I think it's actually better to print it for every execution.